test/dynamo/test_torchrec.py

```diff
@@ -0,0 +1,204 @@
+# Owner(s): ["module: dynamo"]
+import sys
+import unittest
+from typing import Dict, List
+
+import torch
+import torch._dynamo.test_case
+from torch import nn
+
+from torch._dynamo.test_case import TestCase
+from torch._dynamo.testing import CompileCounter
+from torch.testing._internal.common_utils import NoTest
+
+try:
+    from torchrec.datasets.random import RandomRecDataset
```

Comment:
in automated testing suite, my assumption is torchrec is rarely (never) available?

Reply:
In previous PRs are made torchrec get installed for benchmark runs, but on further reflection I probably need to get it installed for test runs too

Reply:
@ezyang  sg.

 If this is something we want to test automatically, its more motivation to get prebuilt fbgemm (potentially through nova) cc @osalpekar , WDYT?

Reply:
pre-built fbgemm would be quite profitable but difficult, as source changes to pytorch will invalidate fbgemm binaries